src/main/src/main/java/org/geoserver/security/decorators/DecoratingWMSLayerInfo.java

```diff
@@ -230,6 +230,4 @@ public boolean isAdvertised() {
     public void setAdvertised(boolean advertised) {
         delegate.setAdvertised(advertised);
     }
-
```

Comment:
Pure formatting change, please avoid (here and in the other files too, it also makes the pull request look bigger than it is)